ev3dev2/sensor/lego.py

```diff
@@ -626,16 +626,32 @@ def reset(self):
         self._ensure_mode(self.MODE_GYRO_ANG)
         self._direct = self.set_attr_raw(self._direct, 'direct', 17)
 
-    def wait_until_angle_changed_by(self, delta):
+    def wait_until_angle_changed_by(self, delta, honor_direction=False):
```

Comment:
I think "honor" might be confusing to non-native English speakers; could we call it something else? Maybe `direction_sensitive`? `specific_direction`?

Reply:
``direction_sensitive`` sounds good